engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/videoSettings/VideoSettingsScreen.java

```diff
@@ -99,18 +130,18 @@ public void initialise() {
             blur.setOptions(Lists.newArrayList(0, 1, 2, 3));
             blur.bindSelection(BindHelper.bindBeanProperty("blurIntensity", config.getRendering(), Integer.TYPE));
             blur.setOptionRenderer(new StringTextRenderer<Integer>() {
-
+            //similar format for the translate string
                 @Override
                 public String getString(Integer value) {
                     switch (value) {
                         case 1:
-                            return "Some";
+                            return translationSystem.translate("${engine:menu#" + "camera-blur-some" + "}");
                         case 2:
-                            return "Normal";
+                            return translationSystem.translate("${engine:menu#" + "camera-blur-normal" + "}");
                         case 3:
-                            return "Max";
+                            return translationSystem.translate("${engine:menu#" + "camera-blur-max" + "}");
                         default:
-                            return "Off";
+                            return translationSystem.translate("${engine:menu#" + "camera-blur-off" + "}");
```

Comment:
Same here - please try to find a general solution that can be applied to all `UIDropdown` boxes. Maybe use an enum?